pkg/apis/kops/componentconfig.go

```diff
@@ -520,6 +520,22 @@ type KubeControllerManagerConfig struct {
 	KubeAPIQPS *float32 `json:"kubeAPIQPS,omitempty" flag:"kube-api-qps"`
 	// KubeAPIBurst Burst to use while talking with kubernetes apiserver. (default 30)
 	KubeAPIBurst *int32 `json:"kubeAPIBurst,omitempty" flag:"kube-api-burst"`
+	// The number of deployment objects that are allowed to sync concurrently.
+	ConcurrentDeploymentSyncs *int32 `json:"concurrentDeploymentSyncs,omitempty" flag:"concurrent-deployment-syncs"`
+	// The number of endpoint objects that are allowed to sync concurrently.
+	ConcurrentEndpointSyncs *int32 `json:"concurrentEndpointSyncs,omitempty" flag:"concurrent-endpoint-syncs"`
+	// The number of namespace objects that are allowed to sync concurrently.
+	ConcurrentNamespaceSyncs *int32 `json:"concurrentNamespaceSyncs,omitempty" flag:"concurrent-namespace-syncs"`
+	// The number of replicaset objects that are allowed to sync concurrently.
+	ConcurrentReplicasetSyncs *int32 `json:"concurrentReplicasetSyncs,omitempty" flag:"concurrent-replicaset-syncs"`
+	// The number of service objects that are allowed to sync concurrently.
+	ConcurrentServiceSyncs *int32 `json:"concurrentServiceSyncs,omitempty" flag:"concurrent-service-syncs"`
+	// The number of resourcequota objects that are allowed to sync concurrently.
+	ConcurrentResourceQuotaSyncs *int32 `json:"concurrentResourceQuotaSyncs,omitempty" flag:"concurrent-resource-quota-syncs"`
+	// The number of serviceaccount objects that are allowed to sync concurrently to create tokens.
+	ConcurrentServiceaccountTokenSyncs *int32 `json:"concurrentServiceaccountTokenSyncs,omitempty" flag:"concurrent-serviceaccount-token-syncs"`
+	// The number of replicationcontroller objects that are allowed to sync concurrently.
+	ConcurrentRcSyncs *int32 `json:"concurrentRcSyncs,omitempty" flag:"concurrent-rc-syncs"`
```

Comment:
this flag does not exist anymore?

Reply:
It actually is still used: https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/

Reply:
Thats strange. It looks like the flag uses underscores rather than hyphens: https://github.com/kubernetes/kubernetes/blob/c48ed1c5aca6e06b6a3d70d5f189216df3150379/cmd/kube-controller-manager/app/options/replicationcontroller.go#L36 

I think you'll need to update this to `flag:"concurrent_rc_syncs"`

Reply:
umm, the k8s cli normalizes hiphens and underscores. So this shouldn't be a problem. https://github.com/kubernetes/kubernetes/blob/release-1.16/staging/src/k8s.io/component-base/cli/flag/flags.go#L36

Reply:
It looks like that normalization may have been introduced in k8s 1.14. On k8s 1.13 I get `unknown flag: --concurrent-rc-syncs` though given that this PR will most likely not be backported to Kops 1.13, I think keeping it with hyphens should be fine, but perhaps add a comment mentioning that it will only work on k8s >= 1.14

Reply:
Will do, makes sense. The normalization might have probably just been missed for rc syncs which seems to be deprecating anyways. Will add the comment.